src/ThreeJsPanel.ts

```diff
@@ -97,6 +108,20 @@ export class ThreeJsPanel {
     }
 
     this.scene = new Scene();
+    this.meshRenderTarget = new WebGLRenderTarget(this.canvas.width, this.canvas.height, {
+      minFilter: NearestFilter,
+      magFilter: NearestFilter,
+      format: RGBAFormat,
+      type: FloatType,
```

Comment:
is float type really what is wanted here?  does this cause it to create a RGBA32F buffer?  For this basic mesh-rasterizing renderer probably RGBA8U is plenty.

Reply:
After a bit more research, it looks like this is possible, but requires an update to three.js to pick up [this change](https://github.com/mrdoob/three.js/pull/22351)